src/constants/links.js

```diff
@@ -3,38 +3,44 @@ export const header = {
     // first dropdown start
     [
       {
-        label: 'navigation.dropdowns.0.linkLabels.0',
-        link: 'link-one',
+        LABEL: 'navigation.dropdowns.0.linkLabels.0',
+        LINK: 'link-one',
       },
       {
-        label: 'navigation.dropdowns.0.linkLabels.1',
-        link: 'link-two',
+        LABEL: 'navigation.dropdowns.0.linkLabels.1',
+        LINK: 'link-two',
       },
     ],
     // first dropdown end
     // second dropdown start
     [
       {
-        label: 'navigation.dropdowns.1.linkLabels.0',
-        link: 'link-three',
+        LABEL: 'navigation.dropdowns.1.linkLabels.0',
+        LINK: 'link-three',
       },
       {
-        label: 'navigation.dropdowns.1.linkLabels.1',
-        link: 'link-four',
+        LABEL: 'navigation.dropdowns.1.linkLabels.1',
+        LINK: 'link-four',
       },
     ],
     // second dropdown end
   ],
-  parentLinks: [{ label: 'navigation.parentLinkLabels.0', link: 'link-five' }], // parent (standalone) links
+  parentLinks: [{ LABEL: 'navigation.parentLinkLabels.0', LINK: 'link-five' }], // parent (standalone) links
 };
 
+export const landing = {
+  HERO_BUTTON_LINK: 'https://github.com/nyc-cto/LPaaS2/',
+  SECTION_BUTTON_LINK: 'https://www1.nyc.gov/assets/cto/#/contact',
+};
+
+
 export const footer = {
   primary: header.navDropDowns
     .flat() // turn nested array into flat/unnested array
     .concat(header.parentLinks), // add parent links (taken directly from header; don't make change here)
   secondary: {
-    terms: 'terms',
-    privacy: 'privacy',
+    TERMS: 'terms',
```

Comment:
Should these also be objects with link and label?

Reply:
It's not as difficult to find the labels for the term and privacy compared to the dropdowns in the translation files, so this is fine.